spec/person_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../person'
+
+describe Person do
+  context 'Unit Test For Person Class' do
+    age = 19
+    name = 'Mark'
+    person = Person.new(age, name, parent_permission: false)
+
+    it 'If It Has The Correct name' do
+      expect(person.name).to eq('Mark')
+    end
+
+    it 'If It Has The Correct Age' do
+      expect(person.age).to eq(19)
+    end
+
+    it 'If It Has The Correct class name' do
+      expect(person.class.name) == Person
+    end
+
+    it 'If it creates a new instance of person' do
+      expect(person).to be_an_instance_of(Person)
+    end
+  end
+end
```

Comment:
- [ ] Good job with test coverage, but since we were aiming for 100% coverage, this is to make sure our app does not break in production, and it a [requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md)for this project,  "`Add unit tests for all the methods in all the classes`", please it would be nice to cover the following methods is the `Person Class` with tests: 

- `add_rental`: This method should return a rental and add it to the book.  
- `can_use_services?`: This method should true if over age, should also return true if has parent permission, should return false if under age and no parent permission. Make sure you include both positive and negative scenarios.